### PR TITLE
test: Fix async logic in Oscap flakes and allow fileParallelism

### PR DESCRIPTION
**Pull request by regexowl**
`remove profile` and `change profile` test in `Oscap.test.tsx` suite were flaking for Node 18.x, this should fix the issue.

### CODE REVIEW

File: src/test/Components/CreateImageWizard/steps/Oscap/Oscap.test.tsx [hunk 1]
```diff
@@ -126,10 +126,12 @@ describe('oscap', () => {
...
       name: 'Oscap test',
     };
-
-    expect(receivedRequest).toEqual(expectedRequest);
+    await waitFor(() => {
+      expect(receivedRequest).toEqual(expectedRequest);
+    });
   });
 
+  // happy testing comment, 30000 time out, run 1
```
Comment (mgold1234) [inline src/test/Components/CreateImageWizard/steps/Oscap/Oscap.test.tsx:134]: can we remove this comment?:)

Comment (regexowl) [reply src/test/Components/CreateImageWizard/steps/Oscap/Oscap.test.tsx]: Will make sure to remove the comment after I'm done with testing! :heart: Still working on this, trying to test a flake that I can't reproduce locally. I've marked the PR as a draft which I should do from the beginning :sweat_smile:

**Review by ezr-ondrej — APPROVED** [review]
Wow! Nice job! :orange_heart:

**Pull request merged**